lib/metrics/services.js

```diff
@@ -130,7 +130,7 @@ module.exports = {
 	'keen': /^https?:\/\/api\.keen\.io/,
 	'keen-proxy': /https:\/\/keen-proxy\.ft\.com\//,
 	'konstructor': /https?:\/\/konstructor\.ft\.com/,
-	'lantern': /^https?:\/\/api-lantern\.ft\.com\/realtime/,
+	'lantern': /^https?:\/\/api-lantern\.ft\.com\/realtime\/.*/,
```

Comment:
Is `/realtime` the only Lantern API we want to track?

```suggestion
	'lantern': /^https?:\/\/api-lantern\.ft\.com\/.*/,
```

https://github.com/Financial-Times/lantern-v2/blob/master/src/server/server.js seems like we potential might call more.

Reply:
I was going to do that, but I figured there was a reason it was this specific since @debugwand put out a PR for it: https://github.com/Financial-Times/next-metrics/pull/332.

Would it be ok to remove the `realtime` bit @debugwand?

Reply:
yes, I put it in there because the previous version didn't work, I thought it might need to be more specific. This change is ok.